src/util/misc.js

```diff
@@ -13,3 +13,32 @@ export const formatTextWrap = (text, maxLineLength) => {
     }
   }, '');
 };
+
+export const darkModeColors = () => {
```

Comment:
Super nitty, but wouldn't `src/lib/color.js` be a better fit for color related functions? :nerd_face:

Reply:
Thx!